crates/matrix-sdk/src/client/builder/mod.rs

```diff
@@ -443,7 +443,9 @@ impl ClientBuilder {
             let mut client =
                 BaseClient::with_store_config(build_store_config(self.store_config).await?);
             #[cfg(feature = "e2e-encryption")]
-            (client.room_key_recipient_strategy = self.room_key_recipient_strategy.clone());
+            {
```

Comment:
Are any of those parenthesis needed? I think no? Or are you keeping them for future e2e-enrytpion specific additions?

Reply:
I did remove the unneeded parenthesis indeed. They were used as a hack because a cfg guard cannot be applied to a single statement, it has to apply to an expression; I found using braces clearer to express this intent, instead of parenthesis.

Reply:
Oh, TIL.